robots.txt

```diff
@@ -1,3 +1,13 @@
+---
+layout: null
+---
+{%- comment -%}
+This liquid template is necessary to only allow search engine crawling in
+production environment. We don't want staging or long-lived previews on Netlify
+to be indexed by search engines.
+{%- endcomment -%}
+
+{%- if jekyll.environment == 'production' -%}
 User-agent: *
 
 # Docker Engine archives
```

Comment:
Once we have proper redirects, we can probably remove all of these (I _think_ these were added when we were still hosting archives, but they also helped to prevent Google indexing archived URLs it found elsewhere with "new" content, but using the "old" URL - not 100% sure though).